docs/okeps/okep-5494-ovn-kubernetes-mcp-server.md

```diff
@@ -0,0 +1,740 @@
+│                                                                              │
+│ Unified Tool Interface:                                                      │
+│ ┌──────────────────────────────────────────────────────────────────────────┐ │
+│ │ Layer 1 (K8s):    kubectl_*, ovnkube_trace                               │ │
+│ │ Layer 2 (OVN):    ovn_nbctl_*, ovn_sbctl_*, ovn_trace_*                  │ │
+│ │ Layer 3 (OVS):    ovs_ofctl_*, ovs_appctl_*, ovs_vsctl_*, retis_*        │ │
+│ │ Layer 4 (Kernel): ip_*, nft_*, iptables_*, conntrack_*                   │ │
+│ │ Layer 5 (Debug):  tcpdump_*, pwru_*                                      │ │
+│ └──────────────────────────────────────────────────────────────────────────┘ │
+└──────────────────────────────────────────────────────────────────────────────┘
+
+                        │ Present unified interface
+                        ▼
+
+┌──────────────────────────────────────────────────────────────────────────────┐
+│                        LLM (OVN-K8s Expert)                                  │
+└──────────────────────────────────────────────────────────────────────────────┘
+```
+
+Doing multiple MCP Servers has clear advantages like each layer owning their
+toolset, independent development and maintenance, granular RBAC, reusability,
+one server can't affect the other. Given we would need to align with different
+layered products team later on for a fully supported set of servers from their
+side which might take several releases, for the phase1 here, we plan to build
+our own monolithic server that could then be split into multiple servers later
+on. A single unified server is simpler, faster to iterate on.
+
+So our current implementation design looks like this:
+
+```
+┌─────────────────────────────────────────────────────────────────────────────┐
+│                              MCP Client                                     │
+│                        "Debug pod connectivity issues"                      │
+└─────────────────────────────────────────────────────────────────────────────┘
+                                       │
+                                 MCP Protocol
+                                       │
+                                       ▼
+┌──────────────────────────────────────────────────────────────────────────────┐
+│                        OVN-Kubernetes MCP Server                             │
+│                                                                              │
+│  ┌─────────────────┐  ┌─────────────────┐  ┌──────────────────────────────┐  │
+│  │   Kubernetes    │  │  Live Cluster   │  │      Offline Bundle          │  │
+│  │     Layer       │  │   Execution     │  │       Execution              │  │
+│  │                 │  │                 │  │                              │  │
+│  │ • kubectl get   │  │ kubectl exec    │  │ Parse must-gather            │  │
+│  │ • kubectl desc  │  │ oc debug node   │  │ Parse sos-report             │  │
```

Comment:
We don't want any OCP specific tooling right?

Reply:
yea no oc debug node is bad. removing it.